amadeus/shopping/__init__.py

```diff
@@ -1,13 +1,10 @@
-from ._flight_offers import FlightOffers
```

Comment:
are we removing flight_offers too? :astonished:

Reply:
this was leftover from the past :) when we removed the low-fare search